src/component/reducer/Reducer.ts

```diff
@@ -456,9 +459,9 @@ export function dispatchMiddleware(dispatch) {
     switch (action.type) {
       case types.INITIATE: {
         if (action.payload) {
-          const { spectra, ...res } = migrate(action.payload);
-          void SpectraManager.fromJSON(spectra, usedColors).then((data) => {
-            action.payload = { spectra: data, ...res, usedColors };
+          const result = migrate(action.payload);
```

Comment:
@jobo322 

the migration is longer need it here because you do the same in https://github.com/cheminfo/nmr-load-save/blob/b17d8af9a8ed22a0115a2e3f9e5f5f3bec03f9c1/src/reader/readNMRiumObject.ts#L17

Reply:
ups, many thanks